portfolio/src/main/webapp/script.js

```diff
@@ -104,24 +110,38 @@ xhttp.onreadystatechange = function() {
     if (xhttp.responseText.includes('<p>You are logged in!</p>')) {
       var form = document.createElement('form');
       form.setAttribute('method', 'POST');
-      form.setAttribute('action', '/data');
+      form.setAttribute('id', 'comment-form');
+      form.setAttribute('class', 'hidden');
+      form.setAttribute('enctype', 'multipart/form-data');
 
       var text = document.createElement('p');
       text.innerText = 'Enter your feedback here:';
 
       var textArea = document.createElement('textarea');
       textArea.name = 'text-input';
-      textArea.required = '';
+      textArea.required = true;
+
+      const imgText = document.createElement('p');
+      imgText.innerText = 'Upload an image:';
+
+      const imgInput = document.createElement('input');
+      imgInput.type = 'file';
+      imgInput.name = 'image';
+      imgInput.setAttribute('class', 'btn btn-outline-secondary');
 
-      var br = document.createElement('br');
+      const br = document.createElement('br');
+      const br2 = document.createElement('br');
```

Comment:
Can you re-use this br? I.e.

`form.appendChild(br);`
`form.appendChild(br);`

(Not sure if it will work or not).

Reply:
No this doesn't work